babel-preset.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable */
+var path = require('path')
+
+// require.resolve has to be used for the presets and plugins,
+// or Babel will look for them relative to the user's folder
+function r(name) {
+  return require.resolve(name)
+}
+
+module.exports = function() {
+  var plugins = [
+    [r('@babel/plugin-transform-runtime'), {
+      helpers: false,
+      polyfill: false,
+      regenerator: true,
+      // require.resolve can't be used here since @babel/runtime has no index
+      moduleName: path.resolve(__dirname, 'node_modules/@babel/runtime')
+    }]
+  ]
+
+  if (process.env.NODE_ENV === 'development') {
+    plugins.unshift(r('react-hot-loader/babel'))
+  }
+
+  return {
+    presets: [
+      [r('@babel/preset-env'), {
```

Comment:
Maybe re-use the browser list that Autoprefixer uses?

Reply:
![image](https://user-images.githubusercontent.com/4061736/32758332-b4cbf822-c948-11e7-8340-7dd2cb1f4755.png)

talking about this specifically

Reply:
![image](https://user-images.githubusercontent.com/4061736/32758364-e5c4c846-c948-11e7-9354-c520ed329ba3.png)

Could just extract the list from `src/webpack/rules/cssLoader.js` and then use it in both the Babel config and CSS loader config, just to reduce repetition.

Reply:
Thanks for the review @SShrike!
Yeah, I alludes to that in the description, "... super cool feature of `useBuiltIns: 'usage'` in Babel 7." The polyfill part of `target.browsers` is not really working properly for this use case yet, and since ["Without any configuration options, `@babel/preset-env` behaves exactly the same as `@babel/preset-latest`"](https://github.com/babel/babel/tree/master/experimental/babel-preset-env), I thought we could leave it as it is for now.

Reply:
More along those lines, @derekperkins and I have been talking about a plugin that could determine these types of parameters based on google analytics device/browser usage thresholds. Something like, "Enter you GA key, and we'll handle the rest".

Reply:
That sounds really awesome!

Look what else just got opened! https://github.com/MatAtBread/fast-async/issues/46